src/app/modules/wallet/pages/account-voluntary-exit/account-voluntary-exit.component.html

```diff
@@ -0,0 +1,54 @@
+<app-breadcrumb></app-breadcrumb>
+<div class="md:w-2/3">
+    <form [formGroup]="exitAccountFormGroup" (ngSubmit)="confirmation()">
+        <mat-card class="bg-paper">
+            <mat-card-content>
+                <h3 class="text-accent">
+                    Are you sure you want exit these account?
+
+                    Once these accounts are exited, the action cannot be
+                    reverted.
```

Comment:
I think a more suitable wording would be
> Please make sure you understand the consequences of performing a voluntary exit. Once an account is exited, the action cannot be reverted

Reply:
Also add in another sentence: "You will not be able to withdraw your ETH if exited until ETH1 is merged with ETH2, which could happen in over a year"